cmake/external_mkldnn.cmake

```diff
@@ -16,6 +16,11 @@
 
 include(ExternalProject)
 
+# Includes blas 3.8.0 in mkldnn 
+set(NGRAPH_MKLDNN_VERSION "v0.18-rc")
+set(NGRAPH_MKLDNN_SUB_VERSION "2019.0.3.20190125")
+set(NGRAPH_MKLDNN_GIT_TAG "08bd90c")
```

Comment:
Can we use a symbolic tag instead? "v0.18-rc" is the same as  "08bd90c"

Reply:
We should be able to use git to get the hash of v0.18-rc so they are always in sync.